recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2020.03.06" %}
+{% set version = datetime.datetime.utcnow().strftime('%Y.%m.%d.%H.%M.%S') %}
```

Comment:
Do we need seconds?

Reply:
I'm worried that if two PRs get merged fast enough + the inconsistency of the CI pushing the builds through could have two builds in the same minute.

Reply:
:smile: ok